flair/datasets/document_classification.py

```diff
@@ -1541,6 +1541,73 @@ def __init__(self,
         )
 
 
+class GERMEVAL_2018_OFFENSIVE_LANGUAGE(ClassificationCorpus):
+    """
+    GermEval 2018 Task 1 corpus for identification of offensive language.
+    Classifying German tweets into 2 coarse-grained categories OFFENSIVE or OTHER.
+    """
+
+    def __init__(self,
```

Comment:
Could you add one more boolean constructor argument called `fine_grained_classes` (or so) that's set to `False` by default? If set to `True` you could load the fine-grained offense classes instead. 

To do this in the current logic, you'd need to generate separate FastText-formatted files for the fine-grained case.

Reply:
Sure. I added support for the fined-grained case. This way one could use the data from both GermEval 2018 tasks (binary and 4-way classification). As you've mentioned before, this requires creating separate files for both tasks.